dipy/viz/app.py

```diff
@@ -775,6 +792,9 @@ def horizon(tractograms=None, images=None, pams=None,
         Each tuple contains data and affine
     pams : sequence of PeakAndMetrics
         Contains peak directions and spherical harmonic coefficients
+    np_files : sequence of arrays
```

Comment:
``np_files`` seems too generic to me if it is going to be used only for BUAN.

Reply:
But I think you kept that name because it can be used for other visualizations too.
Maybe ``npy_files``?